scripts/curate_basel.py

```diff
@@ -0,0 +1,120 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
```

Comment:
to follow our convention-- note: if you accept these changes you will need to also update the code that calls these arguments
```suggestion
    parser.add_argument("-o", "--path-output",
```